server/server_test.go

```diff
@@ -0,0 +1,276 @@
+package server
+
+import (
+	"bytes"
+	"errors"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/klarna/eremetic"
+	"github.com/klarna/eremetic/config"
+	"github.com/klarna/eremetic/mock"
+	"github.com/klarna/eremetic/scheduler"
+)
+
+func TestServer_AddTask(t *testing.T) {
```

Comment:
I'd prefer to have all tests in the project written with the same framework, ie go convey.

Nothing wrong with normal tests, but for cohesiveness.